scripts/seed-searchengine.js

```diff
@@ -0,0 +1,18 @@
+import {} from '../src/index.js'
+import SearchEngine from '../src/services/elasticsearch/index.js'
+
+async function run() {
```

Comment:
you need to init searchEngine here I guess ?

Reply:
Yeah, it can be. but i thought we might add some other configs later to it.
But if you think that's too much i can merge them